cmd/jobsgowhere.go

```diff
@@ -16,22 +16,23 @@ var supportedRoutes = []string{"POST", "OPTIONS", "DELETE", "GET", "PUT", "PATCH
 // ConfigureRoutes - route definitions
 func ConfigureRoutes(router *gin.Engine, db *sql.DB) {
 	api := router.Group("/api")
+	apiWithAuth := router.Group("/api")
 	jc := job.NewController(db)
-	api.Use(oauth.AuthMiddleware())
+	apiWithAuth.Use(oauth.AuthMiddleware())
```

Comment:
How does this work in Go? is apiWithAuth being called from gin-gonic?

Reply:
The router is configured in `main.go` ... ignore the previous few lines as those were added for the Mock API and should be removed.
https://github.com/jobsgowhere/jobsgowhere/blob/master/main.go#L66